torch/csrc/Stream.cpp

```diff
@@ -0,0 +1,112 @@
+#include <pybind11/pybind11.h>
+#include <torch/csrc/Device.h>
+#include <torch/csrc/THP.h>
+#include <torch/csrc/utils/python_arg_parser.h>
```

Comment:
I'm guessing this was modeled off of cuda/Stream.cpp

Reply:
Yes. A common base Stream for all the devices.